src/app/snapp_test_transaction/.ocamlformat

```diff
@@ -0,0 +1,66 @@
+profile=ocamlformat
```

Comment:
should this be a link, so the format isn't duplicated, which may get out of synch?